scripts/convert_awq_marlin.py

```
@@ -8,16 +8,17 @@
 import os
 import shutil
 
+
 def get_scale_perms():
     scale_perm: List[int] = []
```

Ruff (0.11.9)

Undefined name `List`

(F821)